templates/search/index.html.twig

```diff
@@ -6,21 +6,26 @@
     <h1 class="text-center"> Recherche des {{ role }}s </h1>
     <div class="container-fluid card-search">
 
-        <div class="row no-gutters justify-content-center">
+        <div class="row no-gutters justify-content-around">
             {% for user in users %}
-                <div class="card col-10 col-md-4 col-lg-3 col-xl-2 m-3 p-0">
+                <div class="card col-10 col-md-4 col-lg-3 col-xl-2 m-5 p-0">
                     <img src="{{ user.picture }}" class="card-img-top" alt="image de {{ user.firstname }}">
+
                     <div class="card-body">
                         <div class="star mb-4">
                             <p class="m-0">100</p><i class="fas fa-star ml-2"></i>
                         </div>
                         <h2 class="card-text text-handscript mb-3">{{ user.firstname }}</h2>
+                        <hr>
                         <p class="card-text">Univers préférés : Déco, Gastonomie</p>
```

Comment:
cette ligne ne devrait etre ajoutée que quand vous aurez relié users et univers. Sinon tu affiches de choses en dur qui sont fausses (et en plus tu as mis gas**to**nomie :p